app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/KtxGalleryGridFragment.kt

```diff
@@ -233,6 +246,30 @@ class KtxGalleryGridFragment : BaseFragment(), LifecycleObserver {
         }
     }
 
+    override fun uploadFile(filePath: String) {
+        val json = JSONObject()
+        try {
+            json.put("name", "Sungjun Hong")
```

Comment:
are these all hardcodes? it's not in a test file?

Reply:
This is the demo app area and is to demonstrate how the SDK can be used. I'll replace this with a sample data.

Reply:
i would suggest not using real names and emails yeah

Reply:
yeap. I won't use them!